### ISSUE TITLE
grpc - plugin-topology: disallowing deprecated listchannels.include_private

### ISSUE THREAD

**Issue by JssDWt** [ElementsProject/lightning#7105]
### Issue and Steps to Reproduce
In cln 24.02rc2

When calling `ListChannels` through the grpc api, the following message is logged in the log. 
`plugin-topology: Note: disallowing deprecated listchannels.include_private for \"1/cln:listchannels#80\"`

The channel I'm looking for is not found by the listchannels call. 
Note that there is no option for `include_private` in the grpc `ListChannelsRequest`.

This causes my integration tests to fail on cln 24.02rc2

**Comment by daywalker90** [comment]
If you want to get private channels you need to use listpeerchannels now.

**Comment by daywalker90** [comment]
> > If you want to get private channels you need to use listpeerchannels now.
> 
> I'm looking specifically for the `active` field in `listchannels`. Is there an alternative in `listpeerchannels` that is equivalent? A channel can be in state `CHANNELD_NORMAL` without being active.

no there is not, closest thing is probably `last_stable_connection`

**Comment by JssDWt** [comment]
> no there is not, closest thing is probably `last_stable_connection`

Any chance to re-enable this functionality? Or at least add a replacement for the `active` field?

**Comment by cdecker** [comment]
Not really, the private gossip has been split from the public one, since they are fundamentally different. We should add the `active` flag to the `listpeerchannels` output instead.

**Comment by sangbida** [comment]
This issue is being closed as part of issue cleanup.

**Reason**: About deprecated feature

If you believe this issue should remain open, please comment and we can reopen it.

**Comment by daywalker90** [comment]
The active flag has not been added to listpeerchannels yet so this should probably stay open.
